### ISSUE TITLE
Review InstantILL interface vs accessibility guidelines

### ISSUE THREAD

**Issue by JosephMcArthur** [OAButton/discussion#1130]
Need to meet an AA rating on WCAG standards.

**Comment by JosephMcArthur** [comment]
Got a review back for InstantILL itself. Only minor additions needed. Mark, can you add the below with some sensible copy (will leave that to you):

Issue:
The submit button (the magnifying glass) does not have a label. This is important as a screen reader will have no idea what this item does.
Solution:
Add an aria-label to the link element.

Issue: 
The search form does not describe what it does for screen readers. Placeholder text isn’t always read, so it can’t be relied on.
Solution:
Add an aria-label

**Comment by JosephMcArthur** [comment]
Here is a small one for /requests too (well, I hope it's small, if not don't worry!)

Issue:
There are form elements on the page and they all need a label of some sort. These are:
Filter by status
Filter by user
Filter by keyword
Filter by journal

Solution:
Add an aria-label to each of these elements. Using the placeholder text for the “filter” ones as the aria-label should be sufficient.

**Comment by markmacgillivray** [comment]
Done, on the dev version of the instantill widget and the dev requests page. Will go to live when next pushed. @JosephMcArthur close this if you are happy with it.
